packages/components/src/components/brand/brand.scss

```
@@ -1,25 +1,33 @@
 @use "@db-ui/foundations/build/scss/variables";
+@use "@db-ui/foundations/build/scss/colors";
 
-%brand-display {
-	display: flex;
-	gap: variables.$db-spacing-fixed-md;
-	align-items: center;
+:root {
+	* > :has(.db-brand) {
+		// Deactivate underline on site name being made clickable by added to the link as well
+		text-decoration: none;
+	}
```

```suggestion
a > :has(.db-brand) {
	// Deactivate underline on site name being made clickable by added to the link as well
	text-decoration: none;
}
```

we could be more restrictive on this one and even also don't need to prefix it by `:root` (or I don't understand the reason).